core/jazz_create-serverless-service/config/dev-config.json

```diff
@@ -3,12 +3,16 @@
     "JOB_TOKEN": "{job_token}",
     "API_TOKEN": "{api_token}",
     "SVC_USER": "{ci_user}",
-    "REGION": "{conf-region}",
     "SERVICE_API_URL": "https://{conf-apikey}.execute-api.{conf-region}.amazonaws.com",
     "SERVICE_API_RESOURCE": "/dev/jazz/services",
     "TOKEN_URL": "/dev/jazz/login",
     "SERVICE_USER": "{jazz_admin}",
     "TOKEN_CREDS": "{jazz_admin_creds}",
+    "PRIMARY_DEPLOYMENT_ACCOUNT": {
+      "accountId": "{conf-account}",
```

Comment:
`service-configuration-data-loader.groovy` should be updated to load these values during  deployment.

Reply:
It is already updated . Please see https://github.com/tmobile/jazz/pull/650/files#diff-7c3ca5042104db9b883f027237d7004aR330